### DISCUSSION TITLE
part-ii/2017-138/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#263] [Question comments]
Part II 2017 Statistical Physics: Paper 1, Section II, D 

https://questions.tripos.org/part-ii/2017-138/

**Comment by thenewtonium** [comment]
First part is just notes, second part just massaging $\sum_{-J\leq m \leq J} \exp(\mu_B B m / k_B T)$ into the expression they want...

<details> <summary> Sketches </summary>

You should get $\left\langle\mu_{z}\right\rangle = \mu_B\left[(J+\frac{1}{2})\coth(x(J+\frac{1}{2})) - \frac{1}{2}\coth(x/2)\right]$ 

It is helpful to first note that from the physical meaning of $\left\langle\mu_{z}\right\rangle$, it should be increasing with $B$ and never exceed $\mu_B J$ in modulus.

Large $|x|$ behaviour of $\left\langle\mu_{z}\right\rangle/J$  is essentially immediate (asymptotes to $\mu_B$).

For small $|x|$ behaviour, expand $\coth y \approx \frac{1}{y} + \frac{y}{2}$ [By approximating $\sinh y \approx y$ and $\cosh y \approx 1 + \frac{y^2}{2}$], the first term cancels between the two $\coth$'s and we get $\left\langle\mu_{z}\right\rangle / J \approx \frac{x}{2}(J+1)$ for small $|x|$. In particular, the gradient near $0$ is steeper for larger $J$. 

</details>

<details><summary>Magnetic Susceptibility</summary>

Curie's law is that $\chi \sim T^{-1}$. Noting that high temperature corresponds to small $|x|$, the easiest way to show Curie's law holds is to compute high-temperature $\chi$ using the small $|x|$ approximation developed for the sketches.

For *low* temperatures, you need to use the approximation $\coth y \approx 1+2e^{-2y}$ for large $y$. As $J>0$[^first] we can neglect the $e^{-2x(J+\frac{1}{2})}$ term as it $\to 0$ faster than the $e^{-x}$ term, so get $\chi \approx \frac{\mu_B ^2 N}{2k_B T} e^{-x}$.[^second]

For the physical interpretation of the exponential suppression of $\chi$ at low temp., note that for a magnetic system, a low temperature state tends to be one in which most atoms are aligned in the same direction. Hence $M_z$ is *already* almost at its maximum value $J\mu_B N$, so increasing the strength of magnetic field has very little effect.

[^first]: (otherwise $\left\langle\mu_{z}\right\rangle \equiv 0$ anyway)
[^second]: At least, for $x>0$; for $x<0$ you can use that $\chi$ has to be even in $B$ and odd in $T$ (think about it).
</details>
